tests/falco/events.json

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
This makes sense! What about putting this under a `tests/data/outputs` directory, and masking this through `run.NewStringFileAccessor`? That would allow the framework to carry around the test file in a runner-agnostic way (it will work whether Falco runs in a container or in the local machine). Accordingly, I would also add the deserialization code in the same package (the part of code responsible of converting the JSON in a Go struct). Take a look at how some of the existing test data is handled.

Reply:
Sure, will do